### PR TITLE
Phone number assignment cmdlet update

### PR DESCRIPTION
**Pull request by JensTrier**
Added more descriptions to the 2 pages and fixed a typo

### CODE REVIEW

File: teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Accept wildcard characters: False
 ```
 
 ### -PhoneNumber
-The phone number to unnassign from the user or resource account. Supports E.164 format and non-E.164 format.
+The phone number to unassign from the user or resource account. Supports E.164 format and non-E.164 format. Needs to be without prefixed "tel:".
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md:70]:
```suggestion
The phone number to unassign from the user or resource account. Supports E.164 format and non-E.164 format. Needs to be without the prefixed "tel:".

```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 2]
```diff
@@ -31,7 +31,9 @@ Set-CsPhoneNumberAssignment -Identity <String> -EnterpriseVoiceEnabled <Boolean>
 ```
 
 ## DESCRIPTION
-This cmdlet assigns a phone number to a user or resource account.
+This cmdlet assigns a phone number to a user or resource account. When you assign a phone number the EnterpriseVoiceEnabled flag is automatically set to True.
+
+To remove a phone number from a user or resource account use the cmdlet [Remove-CsPhoneNumberAssignment](Remove-CsPhoneNumberAssignment.md)
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:36]:
```suggestion
To remove a phone number from a user or resource account, use the [Remove-CsPhoneNumberAssignment](Remove-CsPhoneNumberAssignment.md) cmdlet.

```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 3]
```diff
@@ -117,8 +125,8 @@ Accept wildcard characters: False
 ```
 
 ### -PhoneNumber
-The phone number to assign to the user or resource account. Supports E.164 format like +12065551234 and non-E.164 format like 12065551234. We are also supporting
-Direct Routing numbers with extensions using the formats +1206555000;ext=1234 or 1206555000;ext=1234
+The phone number to assign to the user or resource account. Supports E.164 format like +12065551234 and non-E.164 format like 12065551234. The phone number can not have
+"tel:" prefixed. We are supporting Direct Routing numbers with extensions using the formats +1206555000;ext=1234 or 1206555000;ext=1234
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:129]:
```suggestion
"tel:" prefixed. We support Direct Routing numbers with extensions using the formats +1206555000;ext=1234 or 1206555000;ext=1234.

```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 4]
```diff
@@ -134,7 +142,8 @@ Accept wildcard characters: False
 ```
 
 ### -PhoneNumberType
-The type of phone number to assign to the user or resource account. The supported values are DirectRouting, CallingPlan and OperatorConnect.
+The type of phone number to assign to the user or resource account. The supported values are DirectRouting, CallingPlan and OperatorConnect. When you acquire a phone number
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:145]:
```suggestion
The type of phone number to assign to the user or resource account. The supported values are DirectRouting, CallingPlan, and OperatorConnect. When you acquire a phone number

```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 5]
```diff
@@ -161,5 +170,11 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## NOTES
 The cmdlet is available in Teams PS module 2.6.1-preview or later.
 
+If a user or resource account has a phone number set in Active Directory on-premises and synched into Microsoft 365, you can't use Set-CsPhoneNumberAssignment to set the phone
+number. You will have to clear the phone number from the on-premises Active Directory and let that change sync into Microsoft 365 first.
+
+The previous command for assigning phone numbers to users Set-CsUser had the parameter HostedVoiceMail. Setting HostedVoiceMail for Microsoft Teams users is no longer
+necessary and that is why the parameter is not avaiable on Set-CsPhoneNumberAssignment.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:177]:
```suggestion
necessary and that is why the parameter is not available on Set-CsPhoneNumberAssignment.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

Thanks & Done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by JensTrier** [comment]
@yogkumgit I would like to add the following info to the Outputs section with two examples. Can I do it in this PR or should I create a new one?

On successful execution of the command no result object is present. On failed execution of the command, a result object is present with a Code string parameter to indicate the error code and a Message string parameter with the additional error or warning message.

**Comment by dariomws** [comment]
Hi @JensTrier, you can do it in this PR while it's still open.

**Comment by JensTrier** [comment]
I've now added text to description section in both cmdlet files and added a new example to Set-CsPhoneNumberAssignment show the result object. Please have a look

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
@JohanFreelancer9 @tseward @get-itips, please review the latest addition.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**